### DISCUSSION TITLE
Howto test responseTime ?

### DISCUSSION THREAD

**Discussion by wiquanAppD** [usebruno/bruno#523] [Q&A]
Is there a way to test the responseTime ?

```
  test("ResponseTime < 1500ms", function() {
    expect(res.responseTime).to.be.below(1500);
  });
```

**Comment by phlanglois** [comment]
@wiquanAppD hello 👋 
yes, your method works in single request tests, but not in runner / asserts because of https://github.com/usebruno/bruno/issues/1145 which is fixed but not yet merged: https://github.com/usebruno/bruno/pull/1276
